scripts/gcs/a6000/create_symlinks.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+ln -s -f $PWD/startup_picture_serve_and_take.sh /etc/init.d/startup_picture_serve_and_take.sh
```

Comment:
Since $PWD includes a relative path, you need to add a README.md file including installation instructions.